kratos/mpi/includes/mpi_data_communicator.h

```diff
@@ -70,6 +70,10 @@ std::vector<type> SendRecv(const std::vector<type>& rSendValues,
 void SendRecv(                                                                               \
     const std::vector<type>& rSendValues, const int SendDestination, const int SendTag,      \
     std::vector<type>& rRecvValues, const int RecvSource, const int RecvTag) const override; \
+void Send(const std::vector<type>& rSendValues,                                              \
```

Comment:
this is just a curiosity, and out of the scope of this PR, but it came to my mind that the mechanism used by boost python to extend the classes might be useful to avoid this macro.